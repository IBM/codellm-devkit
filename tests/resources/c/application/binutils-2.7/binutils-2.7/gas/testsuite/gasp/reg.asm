

foo	.REG	(r1)
	add	foo,foo

bar	.reg	(r2)
	add	bar,foo

	.END
