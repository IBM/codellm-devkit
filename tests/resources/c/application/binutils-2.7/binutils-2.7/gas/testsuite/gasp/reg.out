!

!

!foo	.REG	(r1)
!	add	foo,foo
	add	r1,r1
!

!bar	.reg	(r2)
!	add	bar,foo
	add	r2,r1
!

!	.END
